apps/sbbtimetable/sbb_timetable.star

```diff
@@ -0,0 +1,203 @@
+load("render.star", "render")
+load("http.star", "http")
+load("encoding/base64.star", "base64")
+load("encoding/json.star", "json")
+load("cache.star", "cache")
+load("time.star", "time")
+load("schema.star", "schema")
+
+# Define some constants
+SBB_URL = "https://fahrplan.search.ch/api/stationboard.json?show_delays=1&transportation_types=train&stop="
```

Comment:
Notes: 
- API Docs: https://fahrplan.search.ch/api/help
- Certificate is from R3 and will expire in April. We've had some expire like, two days after we land a new app and cause all sorts of problems 🤦 
- Looks like we'll be rate limited at  10080 departure/arrival tables
- Seems like it's fine to submit queries as long as we are under the req/s limit and I don't see anything that says we need attribution??

Reply:
I can't find anything that says we need attribution: [https://fahrplan.search.ch/api/terms.en.html](https://fahrplan.search.ch/api/terms.en.html)